src/plugins/intel_gpu/src/plugin/compiled_model.cpp

```diff
@@ -48,7 +48,20 @@ CompiledModel::CompiledModel(InferenceEngine::CNNNetwork &network,
         if (config.get_property(ov::internal::exclusive_async_requests)) {
             //exclusiveAsyncRequests essentially disables the streams (and hence should be checked first) => aligned with the CPU behavior
             return executorManager()->getExecutor("GPU");
-        }  else if (config.get_property(ov::num_streams) > 1) {
+        } else if (config.get_property(ov::num_streams) > 1) {
+            if (config.get_property(ov::hint::enable_cpu_pinning)) {
```

Comment:
ov::hint::enable_cpu_pinning  should be in plugin(RW) and CompiledModel (RO) supported list, implement the corresponding R/W code.

Reply:
Update.

Reply:
test case updated.